gemini-resin-archetype/src/main/resources/archetype-resources/src/main/java/DefaultServlet.java

```diff
@@ -0,0 +1,21 @@
+package ${package};
+
+import javax.servlet.annotation.*;
+
+import com.techempower.gemini.transport.*;
+
+@SuppressWarnings("serial")
+@WebServlet(name="DS", urlPatterns="*")
+public class DefaultServlet
+     extends InfrastructureServlet
+{
+  /**
+   * Gets a GeminiApplication object for this application.
+   */
+  @Override
+  public Application getApplication()
+  {
+    return Application.getInstance();
+  }
+
+}
```

Comment:
Actually, `Server` is our default servlet. You should remove this class and just add that `@WebServlet` annotation there.